### ISSUE TITLE
Incorrect number of method arguments in tutorial 3 consumer.js

### ISSUE THREAD

**Issue by meverett** [bakkerthehacker/bramqp#46]
I was using the 3rd tutorial (3 Publish Subscribe) as a starting point to do a basic connection and consume test on a remote broker. I copied the code from consumer.js into my node app and adjusted the socket values, username, password, and vhost to connect to my remote host. The connection worked, but when I would publish a message, nothing was being received in node via the console from line 40 on in consumer.js.

I noticed on a stackoverflow post similar example code but I noticed on the equivalent to line 36 in consumer.js:

```javascript
handle.basic.consume(1, queueName, false, true, true, false, {});
```

...had an extra argument after queueName, a 'null' value. Once I figured out these methods were being declared by the specification XML document, I poked into the specification declared on line 12, 'rabbitmq/full/amqp0-9-1.stripped.extended'.

It declares 8 fields, not 7 like in line 36 listed above. As soon as I added an 8th argument, a 'null' right after the queueName field, I was able to start consuming published messages ok.

Basically, this example doesn't work as-is in my testing, I think it's the wrong number of arguments on line 36 for the declared specification file. This was really confusing to track down as my first experience trying to use the library.

What worked for me was:

```javascript
handle.basic.consume(1, queueName, null, false, true, true, false, {});
```
Thanks!

**Comment by bakkerthehacker** [comment]
Thanks for finding and reporting this! I have looked through all the tutorials and this is the only one that has been missing.  It should be there now.  Sorry about the confusion.
